core/src/main/java/cucumber/runtime/order/OrderType.java

```diff
@@ -0,0 +1,26 @@
+package cucumber.runtime.order;
+
+import java.util.List;
+import java.util.Map;
+
+import gherkin.events.PickleEvent;
+
+public abstract class OrderType {

```

Comment:
I think this class can be turned into an interface. The default implementation for `orderPickleEvents` is only used by `none` order.